client/query/query.go

```diff
@@ -231,7 +231,7 @@ func ConstructPaymentInfo(c *channel.Channel, paid, remaining *big.Int) (Payment
 		status = Open
 	}
 
-	latest, err := getLatestSupported(c)
+	latest, err := getLatestSupportedOrPreFund(c)
```

Comment:
Are we sure we want this behaviour? If there is not yet a supported state, perhaps we should return a `nil` balance or something.

Reply:
I think we should return what makes most sense to a user, which I think might be the existing balance? It would seem odd if you use the RPC client to create a ledger channel, specifying a certain balance, and then the first notification you get has a `nil` balance.

Reply:
We do have the `Proposed` status, so I guess that does make sense. 

I think in general we need to be careful about interpreting states which are not supported, though.